cryptoinvestor/views/assets.py

```diff
@@ -2,9 +2,12 @@
 import json
 
 from cryptoinvestor.views import BaseView
+from flask import Flask, request
 
 
 class AssetsListView(BaseView):
+    methods = ['GET', 'POST']
```

Comment:
Is there any way to use POST request from UI? I don't think this is necessary

Reply:
done